admin_guide/install/advanced_install.adoc

```diff
@@ -197,6 +401,101 @@ node2.example.com         kubernetes.io/hostname=node2.example.com,region=primar
 ----
 ====
 
+*Multiple etcd Hosts*
+
+If you installed multiple *etcd* hosts:
+
+. On a master host, verify the *etcd* cluster health, substituting for the FQDNs
+of your *etcd* hosts in the following:
++
+====
+----
+# etcdctl -C \
+    https://etcd1.example.com:2379,https://etcd2.example.com:2379,https://etcd3.example.com:2379 \
+    --ca-file=/etc/openshift/master/master.etcd-ca.crt \
+    --cert-file=/etc/openshift/master/master.etcd-client.crt \
+    --key-file=/etc/openshift/master/master.etcd-client.key cluster-health
+----
+====
+
+. Also verify the member list is correct:
++
+====
+----
+# etcdctl -C \
+    https://etcd1.example.com:2379,https://etcd2.example.com:2379,https://etcd3.example.com:2379 \
+    --ca-file=/etc/openshift/master/master.etcd-ca.crt \
+    --cert-file=/etc/openshift/master/master.etcd-client.crt \
+    --key-file=/etc/openshift/master/master.etcd-client.key member list
+----
+====
+
+*Multiple Masters*
+
+If you installed multiple masters:
+
+. On a master host, determine which host is currently running as the active
+master:
++
+----
+# pcs status
+----
+
+. After determining the active master, put the specified host into standby mode:
++
+----
+# pcs cluster standby <host1_name>
+----
+
+. Verify the master is now running on another host:
++
+----
+# pcs status
+----
+
+. After verifying the master is running on another node, re-enable the host on standby for normal operation by running:
++
+----
+# pcs cluster unstandby <host1_name>
+----
+
+Red Hat recommends that you also verify your installation by consulting the
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/High_Availability_Add-On_Reference/index.html[High
+Availability Add-on for Red Hat Enterprise Linux documentation].
```

Comment:
@adellape Does the "Red Hat recommends..." but cause a problem? It's available in Ent and Orig docs, right? It might be easier to just say "You can verifiy your installation by...".